opengrok-indexer/src/main/java/org/opengrok/indexer/web/Util.java

```diff
@@ -1196,31 +1196,31 @@ public static boolean dump(Writer out, File file, boolean compressed) {
      * {@link IOException}s get caught and logged, but not re-thrown.
      * @param out dump destination
      * @param file file to dump
-     * @param compressed if {@code true} the denoted file is assumed to be
-     * gzipped
+     * @param compressed if {@code true} the denoted file is assumed to be gzipped
      * @param contextPath an optional override of "/source/" as the context path
      * @return {@code true} on success (everything read and written)
      * @throws NullPointerException if a parameter is {@code null}.
      */
-    public static boolean dumpXref(Writer out, File file, boolean compressed,
-            String contextPath) {
+    public static boolean dumpXref(Writer out, File file, boolean compressed, String contextPath) {
+
         if (!file.exists()) {
             return false;
         }
-        /**
+
+        /*
          * For backward compatibility, read the OpenGrok-produced document
          * using the system default charset.
          */
-        try (InputStream iss = new BufferedInputStream(
-                new FileInputStream(file))) {
-            Reader in = compressed ? new InputStreamReader(new GZIPInputStream(
-                iss)) : new InputStreamReader(iss);
-            dumpXref(out, in, contextPath);
-            return true;
+        try (InputStream iss = new BufferedInputStream(new FileInputStream(file))) {
```

Comment:
You can actually combine multiple resources in try-with-resources and then they will be closed in the reverse order, e.g.:
```java
try (var iss = ...; var in = ...(iss)) {
}
```
This should help with indentation and might be a little clearer.

Reply:
yep, was too lazy to do it.